### PR TITLE
math-magicians: Components

### PR DESCRIPTION
**Pull request by amare1990**
- Created Component directory.
- Created Calculator.js file for the UI of the Calculator component.
- Added styles.css file for styling the Calculator component.
- Fixed linters errors.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* const calcHtmlElem = parser.parseFromString(calcStr, 'text/html').body.firstChild;
+calcWrapper.append(calcHtmlElem); */
+
```
Comment (BertrandConxy) [inline src/components/Calculator.js]: - [ ] **Commented codes should be removed from the codebase in order to keep it clean and therefore easy to read** 😃 👍 This is among the coding best practices. So, please removes these lines of code

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,16 @@
-import logo from './logo.svg';
 import './App.css';
+import parse from 'html-react-parser';
+import Calculator from './components/Calculator';
+import './components/styles.css';
+/* import calcWrapper from './components/Calculator';
+import calcStr from './components/Calculator'; */
```
Comment (BertrandConxy) [inline src/App.js]: - [ ] Please remove these commented codes in order to keep your codebase clean 😃 👍

**Comment by amare1990** [comment]
Hi @Code Reviewers,
Sorry for I requested code review for the 3rd milestone of the project. I wrongly request for the 3rd milestone. Therefore, this pull request is a code review request for the second milestone of the project.  
Sorry for creating the mess again.
Thanks

File: src/App.js [hunk 3]
```diff
@@ -1,27 +1,16 @@
...
+
+const calcObj = new Calculator();
 
 function App() {
   return (
     <div className="App">
```
Comment (BertrandConxy) [inline src/App.js]: - [ ] From the project requirements, **the `App.js` should be a class-based component. Please use the materials and resources provided in order to update this function to a class-based component** 😃 👍

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @amare1990  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️ 

- You made an invalid submission. Please submit this milestone through the right portal 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Lucas-Masaba** [comment]
Hello @amare1990 , no problem. These things happen

**Comment by amare1990** [comment]
> 

Thanks @Lucas-Masaba.

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <div class ="calc-btn op"> - </div>
+        <div class ="calc-btn"> 1 </div>
+        <div class ="calc-btn"> 2 </div>
+        <div class ="calc-btn"> 3 </div>
+        <div class ="calc-btn op"> + </div>
+        <div class ="calc-btn zero-btn"> 0 </div>
+        <div class ="calc-btn"> . </div>
+        <div class ="calc-btn op"> = </div>
+      </div>
+    </div>
```
Comment (BertrandConxy) [inline src/components/Calculator.js]: - [ ] As in the project requirements, the `Calculator.js` component is supposed to be a class-based component. Please use the materials and the resources provided in order to update to the class-based component. Here are some articles that can be helpful 😃 👍 [Article from geeks](https://www.geeksforgeeks.org/reactjs-class-based-components/) and Article from [W3 schools](https://www.w3schools.com/react/react_class.asp)

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @amare1990, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 4 above)
Comment (amare1990) [reply src/components/Calculator.js]: Thanks @Lucas-Masaba for your reference. Now the comments seem to be fixed.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### PS: I really liked your design :+1: 
## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by amare1990** [comment]
Thanks @GzimAsani for you liked my project design.

**Pull request merged**